src/components/EditCoursePage/CollapsibleCourseRunFields.test.jsx

```diff
@@ -19,7 +19,7 @@ const pacingTypeOptions = [
 
 const initialvalues = [
   {
-    start: '2000-01-01T00:00:00Z', // Different format to be transformed
+    start: '2000-01-01T12:00:00Z', // Different format to be transformed
```

Comment:
Change it to 12 so that you don't get different days depending on if you run the test inside or outside the shell